README.md

```diff
@@ -14,6 +14,8 @@ pip install mkdocs
 pip install mkdocs-material
 ```
 
+If youre stuck check out [the offical documentation on how to install mkdocs](https://www.mkdocs.org/#installation).
```

Comment:
Did the guides in this not work? Also, just FYI this README is not rendered on the site anywhere, only on GH. So might be good to include this on the "working with mk docs" page

Reply:
I had issues with windows not wanting to install with pip, I can add it to the working with mkdocs page though

Reply:
Just saw that you requested my review here. Shoot me a slack next time.